### PR TITLE
Add personal information collection framework in `Tournaments/Official_support`

### PR DESCRIPTION
**Pull request by hburn7**
- [x] Tournament Committee approval
- [x] Address outdated translations (need to sync with https://github.com/ppy/osu-wiki/pull/12419)

This PR adds a framework regarding the collection and use of personal information which tournament organizers must abide by. Edits to language and structure are appreciated.

### CODE REVIEW

**Comment by hburn7** [comment]
Updated to use the term 'registrants' instead of 'users' to avoid accidentally lumping in staff members with this change. Staff members are not affected by this rule change. For example, tournament hosts do not need to explain why they are collecting emails from staff members as this is an extremely common practice.

File: wiki/Tournaments/Official_support/en.md [hunk 1]
```diff
@@ -47,6 +47,14 @@ Community-run tournaments which abide by the following hard criteria are eligibl
   - osu!catch: 5,000
   - osu!mania: 40,000
   - This does not apply to open rank tournaments.
+- The tournament organisers do not collect, allow the submission of, or utilize any personal information[^personal-information] from any registrants for any reason outside of these defined exceptions.
+  - Email addresses may be collected and used by tournament organisers for any reason so long as it is necessary for the tournament to function.
+  - Personal information may be collected as needed to fulfill legal, financial, or tax obligations and to fulfill the distribution of monetary or physical prizes.
+  - Other exceptions to this rule may be granted by the osu! team, provided the tournament organiser provides a thorough justificaiton to the account support team via [tournaments@ppy.sh](mailto:tournaments@ppy.sh) well before any such information is collected.
+- In cases where **any** personal information is collected from registrants for any reason, tournament organisers must include the following on the tournament's forum post and/or wiki page:
+  - Any users, organizations, or other entities who can view this information.
```
Comment (hburn7) [inline wiki/Tournaments/Official_support/en.md:55]: Could also be written as "Who can view the information" - open to feedback on the wording.

File: wiki/Tournaments/Official_support/en.md [hunk 2]
```diff
@@ -47,6 +47,14 @@ Community-run tournaments which abide by the following hard criteria are eligibl
   - osu!catch: 5,000
   - osu!mania: 40,000
   - This does not apply to open rank tournaments.
+- The tournament organisers do not collect, allow the submission of, or utilize any personal information[^personal-information] from any registrants for any reason outside of these defined exceptions.
```
Comment (Albionthegreat) [inline wiki/Tournaments/Official_support/en.md:50]:
```suggestion
- The tournament organisers do not collect, allow the submission of, or utilise any personal information[^personal-information] from any registrants for any reason outside of these defined exceptions.
```

File: wiki/Tournaments/Official_support/en.md (hunk 1 above)
Comment (Albionthegreat) [inline wiki/Tournaments/Official_support/en.md:55]:
```suggestion
  - Any users, organisations, or other entities who can view this information.
```

**Review by Albionthegreat — CHANGES_REQUESTED** [review]
UK English. Also addresses a small typo.

**Comment by hburn7** [comment]
These changes have been approved by the Tournament Committee.

File: wiki/Tournaments/Official_support/en.md [hunk 3]
```diff
@@ -47,6 +47,14 @@ Community-run tournaments which abide by the following hard criteria are eligibl
...
   - This does not apply to open rank tournaments.
+- The tournament organisers do not collect, allow the submission of, or utilise any personal information[^personal-information] from any registrants for any reason outside of these defined exceptions.
+  - Email addresses may be collected and used by tournament organisers for any reason so long as it is necessary for the tournament to function.
+  - Personal information may be collected as needed to fulfill legal, financial, or tax obligations and to fulfill the distribution of monetary or physical prizes.
+  - Other exceptions to this rule may be granted by the osu! team, provided the tournament organiser provides a thorough justification to the account support team via [tournaments@ppy.sh](mailto:tournaments@ppy.sh) well before any such information is collected.
+- In cases where **any** personal information is collected from registrants for any reason, tournament organisers must include the following on the tournament's forum post and/or wiki page:
+  - Any users, organisations, or other entities who can view this information.
+  - How the information will be stored and processed.
+  - What the information is used for.
```
Comment (Hiviexd) [inline wiki/Tournaments/Official_support/en.md:57]: consistency with above point
```suggestion
  - What the information will be used for.
```

**Review by Walavouchey — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**